doc/source/structures/collections/range.rst

```diff
@@ -0,0 +1,91 @@
+.. _range:
+
+Range
+=====
+
+:struct:`Range` is a type that represents a sequence of scalar whole numbers. The sequence can start and finish at any whole number, can be either descending or ascending and can skip numbers.
+
+.. note::
+
+    This is one of the few places in kOS where there is a distinction between
+    decimal (floating point, or fractional) scalar numbers and whole (integer,
+    or round) scalar numbers.  Using a decimal scalar will not throw an error,
+    however it may give unexpected results due to rounding.
+
+There are 3 ways of constructing a :struct:`Range`:
+
+- :code:`RANGE(FROM, TO, STEP)`
+  Will create a sequence of numbers that starts with `FROM` and contains all numbers `N + STEP` such that `N` belongs to the sequence and that are smaller than `TO`. For example
```

Comment:
The phrase "that are smaller than" is wrong when counting backward with a negative STEP.  Then the endpoint is at the low end of the range, not the high end.